### PR TITLE
Translated 1.x to Russian

### PR DESCRIPTION
**Pull request by saundefined**
**Summary**

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [x] Docs
- [ ] Build-related changes
- [ ] Other, please describe:

If changing the UI of default theme, please provide the **before/after** screenshot:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)

You have tested in the following browsers: (Providing a detailed version will be better.)

- [x] Chrome
- [ ] Firefox
- [x] Safari
- [ ] Edge
- [ ] IE

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature
- [ ] Related documents have been updated
- [ ] Related tests have been updated

To avoid wasting your time, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**
- Translated 1.x to Russian
- At Getting Started page updated Node.js version the same, as on index page

### CODE REVIEW

File: packages/docs/docs/ru/config/README.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+      '@org/foo', // равно @org/markdown-it-foo, если существует
+      ['markdown-it-bar', {
+        // предоставьте опции здесь
+      }]
+    ]
+  }
+}
+```
+
+иои
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
или
```

File: packages/docs/docs/ru/config/README.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+  }
+}
+```
+
+### markdown.extendMarkdown
+
+- Тип: `Function`
+- По умолчанию: `undefined`
+
+Функция для изменения конфигурации по умолчанию или применения дополнительных плагинов к экземпляру [markdown-it](https://github.com/markdown-it/markdown-it), используемому для визуализации исходных файлов. например
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Функция для изменения конфигурации по умолчанию или добавления других плагинов к [markdown-it](https://github.com/markdown-it/markdown-it), используемому для преобразования исходных файлов. Например
```

File: packages/docs/docs/ru/config/README.md [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+    extendMarkdown: md => {
+      md.set({ breaks: true })
+      md.use(require('markdown-it-xxx'))
+    }
+  }
+}
+```
+
+::: tip
+Эта опция также включена в [Plugin API](../plugin/option-api.md#extendmarkdown).
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Эта опция также используется в [Plugin API](../plugin/option-api.md#extendmarkdown).
```

File: packages/docs/docs/ru/config/README.md [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+      md.use(require('markdown-it-xxx'))
+    }
+  }
+}
+```
+
+## Сборка Pipeline
+
+:::tip Настройка CSS препроцессоров
+VuePress поставляется со встроенным webpack для перечисленных ниже препроцессоров CSS. Для получения дополнительной информации об установке этих или препроцессоров без встроенной поддержки смотрите [Использование препроцессоров](../guide/using-vue.md#испоnьзование-препроцессоров) для получения дополнительной информации.
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
VuePress имеет готовую конфигурацию webpack для перечисленных ниже препроцессоров CSS. Для получения дополнительной информации смотрите [Использование препроцессоров](../guide/using-vue.md#испоnьзование-препроцессоров).
```

File: packages/docs/docs/ru/config/README.md [hunk 5]
```diff
@@ -0,0 +1,385 @@
...
+:::
+
+### postcss
+
+- Тип: `Object`
+- По умолчанию: `{ plugins: [require('autoprefixer')] }`
+
+Параметры для [postcss-loader](https://github.com/postcss/postcss-loader). Обратите внимание, что при указании этого значения автоматический префикс будет перезаписан, и вам нужно будет включить его самостоятельно.
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Параметры для [postcss-loader](https://github.com/postcss/postcss-loader). Обратите внимание, что при указании этого значения `autoprefixer` будет перезаписан, вам нужно будет добавить его самостоятельно.
```

File: packages/docs/docs/ru/config/README.md [hunk 6]
```diff
@@ -0,0 +1,385 @@
...
+- По умолчанию: `{ preferPathResolver: 'webpack' }`
+
+Опции для [stylus-loader](https://github.com/shama/stylus-loader).
+
+### scss
+
+- Тип: `Object`
+- По умолчанию: `{}`
+
+Опции для [sass-loader](https://github.com/webpack-contrib/sass-loader) для загрузки `*.scss` файлов.
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Опции для [sass-loader](https://github.com/webpack-contrib/sass-loader) для обработки `*.scss` файлов.
```

File: packages/docs/docs/ru/config/README.md [hunk 7]
```diff
@@ -0,0 +1,385 @@
...
+
+### sass
+
+- Тип: `Object`
+- По умолчанию: `{ indentedSyntax: true }`
+
+Опции для [sass-loader](https://github.com/webpack-contrib/sass-loader) для загрузки `*.sass` файлов.
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Опции для [sass-loader](https://github.com/webpack-contrib/sass-loader) для обработки `*.sass` файлов.
```

File: packages/docs/docs/ru/config/README.md [hunk 8]
```diff
@@ -0,0 +1,385 @@
...
+
+### less
+
+- Тип: `Object`
+- По умолчанию: `{}`
+
+Опции для [less-loader](https://github.com/webpack-contrib/less-loader).
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Опции для [less-loader](https://github.com/webpack-contrib/less-loader) для обработки `*.less`.
```

File: packages/docs/docs/ru/config/README.md [hunk 9]
```diff
@@ -0,0 +1,385 @@
...
+
+### configureWebpack
+
+- Тип: `Object | Function`
+- По умолчанию: `undefined`
+
+Измените внутренний конфиг webpack. Если значением является Object, оно будет объединено в окончательную конфигурацию с помощью [webpack-merge](https://github.com/survivejs/webpack-merge); Если значение является функцией, оно получит конфигурацию в качестве 1-го аргумента и флаг `isServer` в качестве 2-го аргумента. Вы можете либо изменить конфигурацию напрямую, либо вернуть объект для слияния:
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Изменение внутренних настроек webpack. Если значением является объект, он будет объединен с помощью [webpack-merge](https://github.com/survivejs/webpack-merge). Если значение является функцией, оно получит текущие настройки в качестве 1-го аргумента и флаг `isServer` в качестве 2-го аргумента. Вы можете изменить настройки напрямую, либо вернуть объект для слияния:
```

File: packages/docs/docs/ru/config/README.md [hunk 10]
```diff
@@ -0,0 +1,385 @@
...
+  }
+}
+```
+
+### chainWebpack
+
+- Тип: `Function`
+- По умолчанию: `undefined`
+
+Измените внутреннюю конфигурацию веб-пакета с помощью [webpack-chain](https://github.com/mozilla-neutrino/webpack-chain).
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Изменение внутренних настроек webpack с помощью [webpack-chain](https://github.com/mozilla-neutrino/webpack-chain).
```

File: packages/docs/docs/ru/config/README.md [hunk 11]
```diff
@@ -0,0 +1,385 @@
...
+```
+
+## Совместимость браузеров
+
+### evergreen
+
+- Тип: `boolean`
+- По умолчанию: `false`
+
+Установите значение `true`, если вы ориентируетесь только на вечнозеленые браузеры. Это отключит перенос ES5 и полизаполнения для IE, что приведет к более быстрой сборке и уменьшению размера файлов.
```
Comment (starkovsky) [inline packages/docs/docs/ru/config/README.md]:
```suggestion
Установите значение `true`, если вы хотите поддерживать только актуальные браузеры. Это отключит предварительную обработку в ES5 и добавление polyfills для IE, что приведет к более быстрой сборке и уменьшению размера файлов.
```

**Comment by saundefined** [comment]
@starkovsky Thanks for your review!
I rewrote most of the files.

**Comment by gevera** [comment]
I hope this will be merged soon. Спасибо, @saundefined!

**Comment by starkovsky** [comment]
@gevera It is incorrect translate, just see #1644 
@NataliaTepluhina Google translate has been merged =(

**Comment by sobolevn** [comment]
See https://github.com/vuejs/vuepress/pull/1644#issuecomment-520485544

TLDR: the quality is really low, this needs a complete rewrite. Feel free to reopen or create a new one when all the google-translate-style issues will be fixed. Feel free to ask me for any help you need. 🙂 

I am really interested in creating awesome Russian docs 🇷🇺 for this tool!
